### PR TITLE
[Enhancement] (nereids)implement showProcCommand in nereids

### LINKED ISSUE #42750: [Enhancement] (nereids)implement showProcCommand in nereids
showProc command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following showProc command in nereids:
unsupportedshowProcStatement
``` SHOW PROC path=STRING_LITERAL                                                 ```

### PR DESCRIPTION
**Pull request by Vallishp**
### What problem does this PR solve?

implement show proc in nereids

Issue Number: close #42750 

Related PR: #xxx

Problem Summary:

### Check List (For Committer)

- Test 

    - [ *] Regression test // added
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No colde files have been changed.
        - [ ] Other reason 

- Behavior changed:

    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?

    - [ ] No.
    - [ ] Yes. 

- Release note

    
    
    None

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowProcCommand.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+/**
+ * show proc command
+ */
+public class ShowProcCommand extends ShowCommand {
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowProcCommand.java:45]: pls rebase the latest code and override `RedirectStatus toRedirectStatus(); `method. You can take `getRedirectStatus` method in ShowProcStmt.java as reference

**Comment by Vallishp** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]
LGTM

**Comment by LiBinfeng-01** [comment]
LGTM

**Review by LiBinfeng-01 — APPROVED** [review]

**Pull request merged**